src/sequence_state.h

```diff
@@ -121,7 +144,9 @@ class SequenceStates {
           std::string, const inference::ModelSequenceBatching_State&>&
           state_output_config_map,
       const size_t max_batch_size,
-      const std::unordered_map<std::string, InitialStateData>& initial_state);
+      const std::unordered_map<std::string, InitialStateData>& initial_state,
+      TRITONSERVER_InstanceGroupKind kind, int32_t device_id,
+      const std::map<int, size_t>& cuda_virtual_address_size);
```

Comment:
cuda_vitrual_address_size - is this set per model or per server?

Reply:
discussed offline it is per server.